dfsclient/pom.xml

```diff
@@ -100,12 +100,12 @@
         <dependency>
             <groupId>org.hpccsystems</groupId>
             <artifactId>commons-hpcc</artifactId>
-            <version>1.0.10-SNAPSHOT</version>
+            <version>1.0.12-SNAPSHOT</version>
```

Comment:
I don't think these changes should be part of the commit

Reply:
Without that change, the dfsclient unit tests fail to run because dfs is referencing a version of commons-hpcc that lacks the expected object used by the unit test.  I didn't want to check in a change that made your auto-unit tests fail.